contrib/debian/rules

```diff
@@ -42,7 +42,7 @@ override_dh_auto_configure:
 		export DELL="-Dplugin_dell=false"; \
 	fi; \
 	if pkg-config --exists efivar; then \
-		export UEFI="-Dplugin_uefi=true -Dplugin_redfish=true -Dplugin_nvme=true"; \
```

Comment:
This is a case that we're going to have to deviate behaviors for Ubuntu and Debian.  `secureboot-db` is a Ubuntu package.  I'll push the changes needed for contrib/debian/rules to get this right between them.

Reply:
Thanks.